public/index.html

```diff
@@ -1,27 +1,110 @@
+    <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-app.js"></script>
+    <script
+      defer
+      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-analytics.js"
+    ></script>
+    <script
+      defer
+      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-firestore.js"
+    ></script>
+    <script
+      src="https://kit.fontawesome.com/0d4aacb00e.js"
+      crossorigin="anonymous"
+    ></script>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="manifest" href="manifest.json" />
+    <title>MyCovidConnect: Find Nearest Healthcare facility</title>
+    <style>
+      .add-button {
+        position: absolute;
+        top: 1px;
+        left: 1px;
+      }
+      .add-to-btn {
+        background-color: #f15c08;
+        padding: 10px;
+        border: 0px;
+        color: wheat;
+        border-radius: 20px;
+        font-weight: normal;
+        text-transform: capitalize;
+        font-family: "Lato", sans-serif;
+      }
+    </style>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <div class="add-to">
+      <div style="position: fixed; bottom: 10px; right: 10px; color: green">
+        <button class="add-to-btn">Install Our App</button>
+      </div>
+    </div>
+    <script>
+      if ("serviceWorker" in navigator) {
+        window.addEventListener("load", function () {
+          navigator.serviceWorker.register("/serviceworker.js").then(
+            function (registration) {
+              console.log(
+                "ServiceWorker registration successful with scope: ",
+                registration.scope
+              );
+            },
+            function (err) {
+              console.log("ServiceWorker registration failed: ", err);
+            }
+          );
+        });
+      }
+      let deferredPrompt;
+      var div = document.querySelector(".add-to");
+      var button = document.querySelector(".add-to-btn");
+      div.style.display = "none";
+      window.addEventListener("beforeinstallprompt", (e) => {
+        e.preventDefault();
+        deferredPrompt = e;
+        div.style.display = "block";
+        console.log(`'beforeinstallprompt' event was fired.`);
+        button.addEventListener("click", async () => {
+          div.style.display = "none";
+          deferredPrompt.prompt();
+          const { outcome } = await deferredPrompt.userChoice;
+          console.log(`User response to the install prompt: ${outcome}`);
+          deferredPrompt = null;
+        });
+        window.addEventListener("appinstalled", () => {
+          div.style.display = "none";
+          // Clear the deferredPrompt so it can be garbage collected
+          deferredPrompt = null;
+          // Optionally, send analytics event to indicate successful install
```

Comment:
no need to trigger alert. although this would be a nice to have event trigger, you can add a task here like:
TODO:  send analytics event to indicate successful pwa app install